src/main/java/cosminsechelmain/utils/FileUtils.java

```diff
@@ -0,0 +1,20 @@
+package cosminsechelmain.utils;
```

Comment:
Add this class in the existing cosminshechel package inside utils. No need to create cosminsechelmain package. Thanks!